contracts/OperatorWallet.sol

```diff
@@ -0,0 +1,157 @@
+pragma solidity ^0.8.0;
+
+import "./lib/IMintAndBurnable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/ContextUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/AddressUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/security/ReentrancyGuardUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+contract OperatorWallet is Initializable, ContextUpgradeable, ReentrancyGuardUpgradeable {
+
+    using SafeERC20Upgradeable for IERC20Upgradeable;
+
+    address private _owner;
+    address private _operator;
+    address private _pendingOwner;
+
+    mapping(address => uint256) hotWalletsMap;
+    address[] public hotWallets;
+    uint256 public numOfHotWallets;
```

Comment:
it may be better to remove the storage `numOfHotWallets` and use `hotwallets.length` instead

Reply:
fixed